main.py

```diff
@@ -112,6 +113,7 @@ def main():
 
     neopx = NeoPx(Config.get('neopixel','count'))
     # tts = TTS(translator=translator)
+    emotion_analysis = EmotionAnalysis()
```

Comment:
We'd need to remove this if we want to merge as it will conflict with the current behaviour of the neopixels. Or disable it in a config file.